src/org/zaproxy/zap/utils/FontUtils.java

```diff
@@ -203,4 +245,17 @@ public static float getScale() {
 	public static boolean isDefaultFontSet() {
 		return defaultFontSet;
 	}
+	
+	/**
+	 * Tells whether or not a custom default HTTP message font was set.
+	 * <p>
+	 * If no custom HTTP message font was set it's used the system default font.
+	 *
+	 * @return {@code true} if a custom font was set, {@code false} otherwise.
+	 * @since 2.7.0
```

Comment:
`TODO add version` instead of `2.7.0`, be good other new methods had the since tag.

Reply:
This one is pending.